### ISSUE TITLE
playerberryskill not being updated

### ISSUE THREAD

**Issue by jackcmedia** [Athlon007/MOP#30]
**Describe the bug**
TL;DR: MOP 2.12.2 causes berry picking skill changes to not be reflected on the save file upon saving.

Playing MSC 29.05.2020 with MSCLoader 1.17 and MOP 2.12.2, picking strawberries increases my berry picking skill within the game (takes less picks to clear a bush) but, when saving, the berry picking skill is not written to the savefile and remains the same as before going to pick strawberries.

Issue is **NOT** reproducible playing MSC 29.05.2020, with MSCLoader 1.17 and **WITHOUT** MOP 2.12.2
Issue is **NOT** reproducible playing stock MSC 29.05.2020

**To Reproduce**
1. Load the game with MSCLoader 1.17 and MOP 2.12.2, and start a new run.
2. Save the game, load MSCEditor 1.11 and note the playerberryskill value.
3. Load the game and go to the strawberry field.
4. Pick four bushes clean (becomes more noticeable when picking clean more bushes)
5. Save the game, load MSCEditor 1.11 and note the playerberryskill value. Notice that it remained the same, despite having picked clean four bushes.

**Expected behavior**
Berry picking skill changes should be written to the save file when saving the game.

**Screenshots**
N/A

**Game Version:**
29.05.2020

**Mod Report:**
Modern Optimization Plugin
Version: 2.12.2
MSC Mod Loader Version: 1.1.7
Date and Time: 2020-08-03T18:33:59Z
Windows 7 Service Pack 1 (6.1.7601) 64bit 

=== MOP SETTINGS ===

ActiveDistance: 1
ActiveDistanceMultiplicationValue: 1
SafeMode: False
RemoveEmptyBeerBottles: False
ToggleVehiclePhysicsOnly: False
IgnoreModVehicles: False
EnableFramerateLimiter: False
FramerateLimiter: 60
EnableShadowAdjusting: False
KeepRunningInBackground: True
DynamicDrawDistance: False
ShadowDistance: 200
RulesAutoUpdate: True
RulesAutoUpdateFrequency: 2
CustomRuleFile: False

CheckSteam: True 
ExperimentalBranch: False

=== MODS ===

No other mods found!

=== RULE FILES ===

No rule files loaded!

**Additional context**
output_log.txt: https://pastebin.com/k6bbSE6D

**Comment by Athlon007** [comment]
Fixed in 3.0.
